### PR TITLE
2234 lesmis navigation

### PR DESCRIPTION
**Pull request by tcaiger**
### Changes:

- Update NavBar
- Add MainMenu
- Add placeholder FavouritesMenu

---

### Screenshots:
See issue

### CODE REVIEW

File: packages/lesmis/src/components/FavouritesMenu.js
Comment (tcaiger) [inline packages/lesmis/src/components/FavouritesMenu.js:16]: No need to review this file now as it is just a placeholder

File: packages/lesmis/src/components/SearchBar.js
Comment (tcaiger) [inline packages/lesmis/src/components/SearchBar.js:12]: No need to review this file now as it is just a placeholder

File: packages/lesmis/src/views/AboutView.js
Comment (tcaiger) [inline packages/lesmis/src/views/AboutView.js:1]: No need to review this file now as it is just a placeholder

File: packages/lesmis/src/views/ContactView.js
Comment (tcaiger) [inline packages/lesmis/src/views/ContactView.js:1]: No need to review this file now as it is just a placeholder

File: packages/lesmis/src/components/Layout.js
Comment (tcaiger) [inline packages/lesmis/src/components/Layout.js:22]: No need to review this file now as it is just a placeholder

File: packages/lesmis/src/components/MainMenu.js
Comment (rohan-bes) [inline packages/lesmis/src/components/MainMenu.js:91]: So just to be clear, this means that if the menu drawer is expanded, and the user presses a key other than the `tab` or `shift` keys then it will close the menu?

I'm wondering if this is standard? Would perhaps just pressing the `esc` key to close it be a bit more common?

File: packages/lesmis/src/components/MainMenu.js
Comment (rohan-bes) [inline packages/lesmis/src/components/MainMenu.js:135]: Once again not at all my area, but I feel that maybe this `powered by` component is a bit more front-and-center than I'm used to seeing it? Perhaps if it will aligned to the bottom of the menu component instead of with the other list items?
Also, not sure what our policy is on this, but wondering if https://www.info.tupaia.org might be a bit of a better URL to redirect people to? tupaia.org will just hit them with a login screen immediately...

**Review by rohan-bes — APPROVED** [review]
Looks really good to me, screenshots are beautiful 😍 

Left a few comments, didn't want to hold up approving as they're rather subjective. Hope you don't take my UI/UX criticisms too seriously, my design skills are borderline non-existent 😝

File: packages/lesmis/src/components/MainMenu.js
Comment (tcaiger) [reply packages/lesmis/src/components/MainMenu.js:106]: Haha your ideas are always helpful Rohan! I should mention that there are designs that I'm working off for this so credit for it looking good goes to Dave! It also means that I'm going to defer to his judgement and leave this as is :)

File: packages/lesmis/src/components/MainMenu.js
Comment (tcaiger) [reply packages/lesmis/src/components/MainMenu.js:131]: Same as above, I'm just going to roll with the designs. In terms of the url though, I think you are right that the info website is a better choice

Comment (tcaiger) [reply packages/lesmis/src/components/MainMenu.js:131]: Yes I agree the info website is a better choice for the url. In regards to the design, I'll once again just roll with the designs.

File: packages/lesmis/src/components/MainMenu.js
Comment (tcaiger) [reply packages/lesmis/src/components/MainMenu.js:87]: hmmm now that I look at it in retrospect I'm not too sure why I did that. I actually stole it from the material-ui docs but I agree that it doesn't make too much sense. I have removed the event listener for keypresses and tested to check that 'enter', 'space' and 'esc' work as expected.

**Comment by rohan-bes** [comment]
Yup, all good about deferring to Dave. Consider this PR approved 👍

**Pull request merged**
